test/Integration/modules/edgecore/eventbus/eventbus_suite_test.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"crypto/tls"
+	MQTT "github.com/eclipse/paho.mqtt.golang"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"testing"
+	"time"
+
+	"github.com/kubeedge/kubeedge/test/Integration/utils/common"
+	"github.com/kubeedge/kubeedge/test/Integration/utils/edge"
+)
+
+//context to load config and access across the package
+var (
+	ctx *edge.TestContext
+	cfg edge.Config
+)
+
+//Interface to validate the MQTT connection.
+type Token interface {
+	Wait() bool
+	WaitTimeout(time.Duration) bool
+	Error() error
+}
+
+const (
+	Device_upload            = "$hw/events/upload/#"
```

Comment:
Please note that kubeedge is using Camel-Case for variable and function name, e.g. DeviceUpload,...

Reply:
Comment here and below.